### PR TITLE
WSL

### PR DESCRIPTION
**Pull request by srid**

### CODE REVIEW

File: home.nix [hunk 1]
```diff
@@ -132,6 +110,12 @@ rec {
         h = "himalaya";
       };
       sessionVariables = { };
+
+      # For WSL
+      profileExtra = ''
+        if [ -e /home/srid/.nix-profile/etc/profile.d/nix.sh ]; then . /home/srid/.nix-profile/etc/profile.d/nix.sh; fi # added by Nix installer
```
Comment (Fuzen-py) [inline home.nix:116]: Maybe it would be better to expand `home.homeDirectory` if you have this public with the intent of other people to use it

Comment (srid) [reply home.nix:116]: Yea, that's a good idea. I'd also like `profileExtra` to take effect only when on non-NixOS (as is the case with WSL).

Comment (Fuzen-py) [reply home.nix:116]: yea there has been quite a few issues documented about the current situation including solutions but nothing has changed. (see nix-community/home-manager#2012 ). for now hacky methods will have to be used to this. I dont believe reading environment variables for conditional config options is an option either, as WSL does have environment variables set to show that its WSL.

**Comment by srid** [comment]
I should just drop this and figure out a way to integrate home-manager with flakes.nix on Ubuntu, cf. https://dee.underscore.world/blog/home-manager-flakes/

**Comment by srid** [comment]
Closing in favour of #10

**Pull request closed without merging**